docs/flatcar-linux/packet.md

```diff
@@ -316,6 +316,7 @@ Lokomotive implements support for some [DNS providers](../dns/), if your provide
 | reservation_ids_default | Default hardware reservation ID for nodes not listed in the `reservation_ids` map. | "" | "next-available"|
 | certs_validity_period_hours | Validity of all the certificates in hours | 8760 | 17520 |
 | controller_clc_snippets [[1]](#clc-snippets-limitation) | Controller Container Linux Config snippets | [] | [example](../advanced/customization.md#usage) |
+| tags | Optional details to tag on Packet resources | `[]` | `["CreatedBy-DevOpsTeam"]` |
```

Comment:
I suggest we use `:` as a separator between the "key" and the "value" in Packet tags. The hyphen is less clear semantically IMO.

Reply:
Updated.